### PR TITLE
changed opreq-control label removal implementation

### PR DESCRIPTION
**Pull request by bitscuit**
to use oc patch instead of oc apply because of unknown issue in oc apply causing labels to return after being removed

### CODE REVIEW

File: cp3pt0-deployment/common/utils.sh [hunk 1]
```diff
@@ -1184,14 +1184,18 @@ function update_operator() {
     fi
 
     title "Updating ${sub_name} in namesapce ${ns}..."
+
+    # there is currently a bug with oc apply where the removed labels are added back by
+    # something (either Open Shift or OLM), so need to use patch
+    # "~1" is the escape sequence for "/" character
+    if [ ! -z "$remove_opreq_label" ]; then
+        ${OC} patch sub ${sub_name} --type='json' -p='[{"op": "remove", "path": "/metadata/labels/operator.ibm.com~1opreq-control"}]' || info "Could not patch Subscription to remove label"
```
Comment (Daniel-Fan) [inline cp3pt0-deployment/common/utils.sh:1192]: @bitscuit 
Would we add a namespace and use full resource name for the command
```suggestion
        ${OC} patch subscription.operators.coreos.com ${sub_name} -n ${ns} --type='json' -p='[{"op": "remove", "path": "/metadata/labels/operator.ibm.com~1opreq-control"}]' || warning "Could not patch Subscription ${sub_name} in ${ns} to remove label"
```

**Comment by Daniel-Fan** [comment]
Test on patch command looks good to me
```
➜  ~ oc patch subscription.operators.coreos.com keycloak-operator -n keycloak --type='json' -p='[{"op": "remove", "path": "/metadata/labels/operator.ibm.com~1opreq-control"}]'
subscription.operators.coreos.com/keycloak-operator patched
```

**Review by Daniel-Fan — CHANGES_REQUESTED** [review]

File: cp3pt0-deployment/common/utils.sh (hunk 1 above)
Comment (bitscuit) [reply cp3pt0-deployment/common/utils.sh]: Yes, updated

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**